<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTabWidget" name="mainTab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="mainSystemLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>784</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHerramientas">
    <property name="title">
     <string>Herramientas</string>
    </property>
    <widget class="QMenu" name="menuModo_grafico">
     <property name="title">
      <string>Modo grafico</string>
     </property>
     <addaction name="actionNativo"/>
     <addaction name="actionOpenGL"/>
    </widget>
    <addaction name="menuModo_grafico"/>
    <addaction name="actionOpciones"/>
   </widget>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo_proyecto"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuVentanas">
    <property name="title">
     <string>Ventanas</string>
    </property>
    <addaction name="actionProyecto"/>
    <addaction name="actionModelo"/>
   </widget>
   <widget class="QMenu" name="menuVista">
    <property name="title">
     <string>Vista</string>
    </property>
    <widget class="QMenu" name="menuGrilla">
     <property name="title">
      <string>Grilla</string>
     </property>
     <addaction name="action10_pixeles"/>
     <addaction name="action20"/>
     <addaction name="action50_pixeles"/>
     <addaction name="actionPersonalizar"/>
    </widget>
    <addaction name="menuGrilla"/>
   </widget>
   <widget class="QMenu" name="menuEdicion">
    <property name="title">
     <string>Edici√≥n</string>
    </property>
    <addaction name="actionDeshacer"/>
    <addaction name="actionRehacer"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEdicion"/>
   <addaction name="menuVista"/>
   <addaction name="menuHerramientas"/>
   <addaction name="menuVentanas"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Principal</string>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="ventanaObjetos">
   <property name="windowTitle">
    <string>Explorador</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="margin">
      <number>2</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="twArbolObjetos">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnSearchMode">
         <property name="text">
          <string>Explorar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Objetos</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="margin">
      <number>2</number>
     </property>
     <item>
      <widget class="QToolBox" name="mainToolBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>135</width>
          <height>108</height>
         </rect>
        </property>
        <attribute name="label">
         <string>General</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QToolButton" name="btnAddMatrix">
           <property name="text">
            <string>Matriz de puntos</string>
           </property>
           <property name="icon">
            <iconset resource="imagenes.qrc">
             <normaloff>:/imagenes/matriz.png</normaloff>:/imagenes/matriz.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnAddImage">
           <property name="text">
            <string>Imagen</string>
           </property>
           <property name="icon">
            <iconset resource="imagenes.qrc">
             <normaloff>:/imagenes/manzana-verde-thumb7707762.jpg</normaloff>:/imagenes/manzana-verde-thumb7707762.jpg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnAddRegion">
           <property name="text">
            <string>Regiones</string>
           </property>
           <property name="icon">
            <iconset resource="imagenes.qrc">
             <normaloff>:/imagenes/conjuntos.gif</normaloff>:/imagenes/conjuntos.gif</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>156</width>
          <height>140</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Redes neuronales artificiales</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QToolButton" name="btnAddPerceptron">
           <property name="text">
            <string>Perceptron simple</string>
           </property>
           <property name="icon">
            <iconset resource="imagenes.qrc">
             <normaloff>:/imagenes/perceptron2.png</normaloff>:/imagenes/perceptron2.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnAddADALINE">
           <property name="text">
            <string>Adaline</string>
           </property>
           <property name="icon">
            <iconset resource="imagenes.qrc">
             <normaloff>:/imagenes/adaline.gif</normaloff>:/imagenes/adaline.gif</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnAddHopfield">
           <property name="text">
            <string>Hopfield</string>
           </property>
           <property name="icon">
            <iconset resource="imagenes.qrc">
             <normaloff>:/imagenes/hopfield.jpg</normaloff>:/imagenes/hopfield.jpg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnAddMultilayerPerceptron">
           <property name="text">
            <string>Perceptron multicapa</string>
           </property>
           <property name="icon">
            <iconset resource="imagenes.qrc">
             <normaloff>:/imagenes/artificial neural network.png</normaloff>:/imagenes/artificial neural network.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>145</width>
          <height>288</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Sistemas multiagentes</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QToolButton" name="btnAddAgent">
           <property name="text">
            <string>Agente</string>
           </property>
           <property name="icon">
            <iconset resource="imagenes.qrc">
             <normaloff>:/imagenes/Agent_icon.png</normaloff>:/imagenes/Agent_icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNativo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Nativo</string>
   </property>
   <property name="toolTip">
    <string>Activa el modo grafico nativo</string>
   </property>
  </action>
  <action name="actionOpenGL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>OpenGL</string>
   </property>
   <property name="toolTip">
    <string>Activa el modo grafico OpenGL para renderizacion avanzada</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpciones">
   <property name="text">
    <string>Opciones</string>
   </property>
  </action>
  <action name="actionProyecto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Proyecto</string>
   </property>
  </action>
  <action name="actionModelo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Objetos</string>
   </property>
  </action>
  <action name="actionNuevo_proyecto">
   <property name="text">
    <string>Nuevo proyecto...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action10_pixeles">
   <property name="text">
    <string>10 pixeles</string>
   </property>
  </action>
  <action name="action20">
   <property name="text">
    <string>20 pixeles</string>
   </property>
  </action>
  <action name="action50_pixeles">
   <property name="text">
    <string>50 pixeles</string>
   </property>
  </action>
  <action name="actionPersonalizar">
   <property name="text">
    <string>Personalizar...</string>
   </property>
  </action>
  <action name="actionDeshacer">
   <property name="text">
    <string>Deshacer</string>
   </property>
  </action>
  <action name="actionRehacer">
   <property name="text">
    <string>Rehacer</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSalir</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuSalirClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNativo</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuNativoClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenGL</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuOpenGLClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNuevo_proyecto</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onNewProyectTriggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>menuSalirClick()</slot>
  <slot>menuOpenGLClick()</slot>
  <slot>menuNativoClick()</slot>
  <slot>menuOpcionesClick()</slot>
  <slot>menuAgregarAgenteClick()</slot>
  <slot>btnAgregarAgenteClick()</slot>
  <slot>btnBorrarAgenteClick()</slot>
  <slot>menuContextualListaObjetos(QPoint)</slot>
  <slot>onNewProyectTriggered(bool)</slot>
 </slots>
</ui>
